plugin/virustotal/vtreport.py

```diff
@@ -0,0 +1,404 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = 'gerardofn@virustotal.com'
+
+from datetime import datetime
+import logging
+import vt
+
+SUPPORTED_FILES = ('PEEXE', 'PEDLL', 'MZ', 'MSI', 'COM',
+                   'COFF', 'ELF', 'LINUX', 'MACHO')
+
+
+class VTReport(object):
+
+  def __init__(self, apikey, file_hash, privileges):
+    global SUPPORTED_FILES
+    client = vt.Client(apikey)
+    general = client.get_object('/files/{}', file_hash)
+    file_type = general.get('type_tag')
+    self.private_api = privileges
```

Comment:
Please rename as many of these to self._xxx as possible, to clarify which are private and which are public.